modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -56,7 +61,90 @@ test.describe('Manage object fields through Model Builder', () => {
 		);
 
 		await apiHelpers.listTypeAdmin.deleteListTypeDefinition(
-			ListTypeDefinition.id
+			listTypeDefinition.id
+		);
+	});
+
+	test('all picklist definitions are listed during object field creation', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const listTypeDefinitions = await Promise.all(
+			Array(22)
+				.fill(null)
+				.map(() =>
+					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
+				)
 		);
+
+		createdEntities.listTypeDefinitionIds = listTypeDefinitions.map(
+			({id}) => id
+		);
+
+		const objectDefinition = [
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default'),
+		];
+
+		objectDefinition.map((objectDefinition) =>
+			createdEntities.objectDefinitionIds.push(objectDefinition.id)
+		);
+
+		await page.goto('/');
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
+
+		await viewObjectDefinitionsPage.viewInModelBuilder();
+
+		await modelBuilderPage.openNewFieldModal(objectDefinition[0].name);
+
+		await modelBuilderPage.fillNewObjectFieldLabel(
+			'objectFieldLabel' + getRandomInt()
+		);
+
+		await modelBuilderPage.selectNewObjectFieldBusinessTypeOption(
+			'Picklist'
+		);
+
+		await modelBuilderPage.newObjectFieldSelectPicklist.click();
+
+		const listTypeDefinitionBox =
+			modelBuilderPage.page.getByRole('listbox');
+
+		await expect(listTypeDefinitionBox).toBeVisible();
+
+		await expect(listTypeDefinitionBox.getByRole('listitem')).toHaveCount(
+			22
+		);
+	});
+
+	test.afterEach(async ({apiHelpers}) => {
```

Comment:
Can you take a look at other tests? I believe `afterEach` hooks are placed before `test.describe`.

Reply:
You're 100% right.